pkg/storage/chunk/cache/instrumented.go

```diff
@@ -64,35 +64,35 @@ type instrumentedCache struct {
 	requestDuration                   *instr.HistogramCollector
 }
 
-func (i *instrumentedCache) Store(ctx context.Context, keys []string, bufs [][]byte) {
+func (i *instrumentedCache) Store(ctx context.Context, keys []string, bufs [][]byte) error {
 	for j := range bufs {
 		i.storedValueSize.Observe(float64(len(bufs[j])))
 	}
 
 	method := i.name + ".store"
-	_ = instr.CollectedRequest(ctx, method, i.requestDuration, instr.ErrorCode, func(ctx context.Context) error {
+	return instr.CollectedRequest(ctx, method, i.requestDuration, instr.ErrorCode, func(ctx context.Context) error {
 		sp := ot.SpanFromContext(ctx)
 		sp.LogFields(otlog.Int("keys", len(keys)))
-		i.Cache.Store(ctx, keys, bufs)
-		return nil
+		err := i.Cache.Store(ctx, keys, bufs)
+		return err
```

Comment:
```suggestion
		return i.Cache.Store(ctx, keys, bufs)
```
should work as well :slightly_smiling_face: